Base/Logic/CMakeLists.txt

```diff
@@ -195,18 +195,32 @@ install(TARGETS ${lib_name}
 if(VTK_WRAP_PYTHON)
   include(vtkMacroKitPythonWrap)
 
+  set(_python_wrapped_libraries
+    )
+
+  if(${VTK_VERSION} VERSION_LESS "8.90")
+    list(APPEND _python_wrapped_libraries
+      MRMLLogicPythonD
+      )
+  endif()
+
   vtkMacroKitPythonWrap(
     KIT_NAME ${lib_name}
     KIT_SRCS ${SlicerBaseLogic_SRCS}
-    KIT_PYTHON_LIBRARIES MRMLLogicPythonD
+    KIT_PYTHON_LIBRARIES ${_python_wrapped_libraries}
     KIT_INSTALL_BIN_DIR ${Slicer_INSTALL_BIN_DIR}
     KIT_INSTALL_LIB_DIR ${Slicer_INSTALL_LIB_DIR}
     )
   # Export target
-  set_property(GLOBAL APPEND PROPERTY Slicer_TARGETS ${lib_name}Python ${lib_name}PythonD)
+  set_property(GLOBAL APPEND PROPERTY Slicer_TARGETS ${lib_name}Python)
```

Comment:
VTK's wrapping macros now export targets already (unless you're skipping that logic…which it appears you are?). But then you're missing the Python properties it tacks onto the modules to indicate what the Python import name of the module is (which allows other projects to make Python code which depends on Slicer modules).

Reply:
Since for wrapping our custom VTK classes, we use our custom module [vtkMacroKitPythonWrap.cmake](https://github.com/Slicer/vtkAddon/blob/support-build-with-vtk89/CMake/vtkMacroKitPythonWrap.cmake) that support both VTK 8.2.0 and VTK >= 8.90, we indeed need to export our Slicer targets.  

> vtk_module_autoinit